src/parser/core/Module.ts

```diff
@@ -58,7 +58,9 @@ interface Hook<T extends Event> {
 
 export default class Module {
 	static dependencies: Array<string | MappedDependency> = []
-	static displayOrder = DISPLAY_ORDER.DEFAULT
+	static displayOrder: number = DISPLAY_ORDER.DEFAULT
```

Comment:
`displayOrder` is completely arbitrary. Multiple modules use values outside the bounds of top and bottom - top and bottom are just "sane" defaults that act within the general scope of the page.

i.e. about/checklist/suggestions are above top so that modules need to be really explicit when they want to break the general flow.